internal/cmd/influxql-decode/v1.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/influxdata/flux/influxql"
+	"github.com/spf13/cobra"
+)
+
+func v1(cmd *cobra.Command, args []string) error {
+	for _, arg := range args {
+		f, err := os.Open(arg)
+		if err != nil {
+			return err
+		}
+
+		var resp influxql.Response
+		if err := json.NewDecoder(f).Decode(&resp); err != nil {
+			return err
+		}
+
+		b := new(bytes.Buffer)
+		for _, res := range resp.Results {
+			for _, s := range res.Series {
+				seriesHeader := fmt.Sprintf("%s,", s.Name)
```

Comment:
In the future we should update this to use `influxdata/line-protocol`.

Reply:
Created an issue https://github.com/influxdata/flux/issues/2566 for this